cpp/dolfinx/la/Vector.h

```diff
@@ -262,7 +262,8 @@ auto squared_norm(const V& a)
 /// Compute the norm of the vector
 /// @note Collective MPI operation
 /// @param x A vector
-/// @param type Norm type (supported types are \f$L^2\f$ and \f$L^\infty\f$)
+/// @param type Norm type (supported types are \f$L^1\f$, \f$L^2\f$
```

Comment:
Best to remove to norm types. The available types can be found via the link to `Norm`.